### DISCUSSION TITLE
Does kube-bench scan for containers running as root?

### DISCUSSION THREAD

**Discussion by huornlmj** [aquasecurity/kube-bench#851] [Questions and Help] [answered]
Does kube-bench scan for containers running as root?

**Comment by mozillazg** [comment, marked as answer]
Yes, kube-bench should be running as root. See also https://github.com/aquasecurity/starboard/pull/354#issuecomment-779415714

**Reply by huornlmj** [reply]
Sorry, perhaps I should elaborate.  When kube-bench scans K8s, is it able to identify _any_ containers that are running as root?  As you know, running containers as root is a security concern.  I posted this because I assumed it would like docker-bench does but I could not find a section showing any checks for containers running as root.

**Reply by mozillazg** [reply]
@huornlmj No, kube-bench does not support this case. It only scans cluster-related configure, for your case, maybe use https://github.com/FairwindsOps/polaris is better?
